circle.yml

```diff
@@ -5,9 +5,6 @@ defaults: &defaults
   docker:
     - image: arcanemagus/atom-docker-ci:stable
   steps:
-    # Restore project state
```

Comment:
This step is _crtical_, it's the _next_ step (the `apt-get update ...` one) that should be removed.

Reply:
@vzamanillo This is why the build is failing.

Reply:
I see now!